luggage_question_answer.install

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+* Renames Q&A Moderator role to QA Moderator.
+*/
+function luggage_question_answer_update_7000() {
```

Comment:
I think this update hook should really be luggage_question_answer_update_7100() since it is an update in the 1.x branch instead of an update across Drupal core versions. https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_update_N/7.x

Also, the version number in the .info file probably should increment to 7.x-1.1 in this update.